src/huggingface_hub/commands/user.py

```diff
@@ -55,6 +55,11 @@ def register_subcommand(parser: ArgumentParser):
         ls_parser.add_argument(
             "--organization", type=str, help="Optional: organization namespace."
         )
+        ls_parser.add_argument(
+            "--use-basic",
+            action="store_true",
+            help="Optional: use Basic authentication",
+        )
```

Comment:
as discussed offline i don't think we need to expose this through the CLI (at least for now)